src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_unicast_transports.cs

```diff
@@ -105,6 +109,23 @@ public Task Handle(MyEvent message, IMessageHandlerContext context)
             }
         }
 
+        static void RegisterPublisherForType(EndpointConfiguration config, string publisherEndpoint, Type eventType)
```

Comment:
should this be an extension method to bus config which we provide in the AT framework?